File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    answer = 0
    
    sum = 0
    mul = 1
    
    for i in range(len(num_list)):
        sum += num_list[i]
        mul *= num_list[i]
        
    if sum ** 2 > mul:
        answer = 1
    else:
        answer = 0
        
    return answer

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        
        int mul_all = 1;
        int sum_square = 0;
        
        for(int i=0; i<num_list.length; i++){
            mul_all *= num_list[i];
            sum_square += num_list[i];
        }
        
        sum_square *= sum_square;
        
        if(mul_all < sum_square){
            answer = 1;
        }
        else{
            answer = 0;
        }
        
        
        return answer;
    }
}